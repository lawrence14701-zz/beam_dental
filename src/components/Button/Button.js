import MaterialUIButton from '@material-ui/core/Button';
import React from 'react';

const Button = (props) => {
    return (
        <MaterialUIButton {...props} />
    )
}

export default Button