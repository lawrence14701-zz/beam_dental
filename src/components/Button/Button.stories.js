import React from "react"
import Button from "./Button"

export default {
  title: "Button",
  component: Button,
  args: { variant: "contained", children: "Button" },
  argTypes: { onClick: { action: "clicked" } },
}

const Template = (args) => <Button {...args} />

export const Primary = Template.bind({})
export const Secondary = Template.bind({})

Primary.args = {
  color: "primary",
  type: "error",
  text: "failed",
}

Secondary.args = {
  color: "secondary",
  type: "success",
  text: "good job",
}
