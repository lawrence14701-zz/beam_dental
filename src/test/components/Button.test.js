import React from "react"
import Button from "../../components/Button/Button"
import "../jestsetup"

test("render a button", () => {
  const wrapper = shallow(<Button />)
  expect(wrapper).toMatchSnapshot()
})
