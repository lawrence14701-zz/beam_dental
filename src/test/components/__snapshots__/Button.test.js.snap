// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a button 1`] = `ShallowWrapper {}`;
